calabash-cucumber/lib/calabash-cucumber/usage_tracker.rb

```diff
@@ -131,26 +147,41 @@ def used_cucumber?
       #
       # Collect a hash of usage info.
       def info
-        {
+
+        allowed = allowed_to_track
+
+        return {} if allowed == "none"
```

Comment:
This method should never be called if allowed == "none".  Maybe I raise an exception.